docs/code_coverage.md

```diff
@@ -1,30 +1,54 @@
-# Execute unit testing and generate a code coverage report
-
-## Prerequisites
-
-You need some additional Python modules to run the unit tests and linting.
+## To check branch stability run the sanity suite as follows
 
 ```bash
-pip install mock pytest pylint pytest-mock pytest-cov
-cd serve
 ./torchserve_sanity.sh
 ```
 
-**The above command executes the following**
+## To run frontend build suite run following command
+
+```bash
+frontend/gradlew -p frontend clean build
+```
 
-* TorchServe frontend build suite which consists of :
+TorchServe frontend build suite consists of :
 
   * checkstyle
   * findbugs
   * PMD
   * UT
+  
+The reports are generated at following path : `frontend/server/build/reports`
 
-* TorchServe backend pytest suite
+## To run backend pytest suite run following command
 
-* torch-model-archive pytest suite
+```bash
+python -m pytest --cov-report html:htmlcov --cov=ts/ ts/tests/unit_tests/
+```
+
+The reports are generated at following path : `htmlcov/`
+
+## To run python linting on `ts` package run following command
+
+```bash
+pylint -rn --rcfile=./ts/tests/pylintrc ts/.;
+```
+
+## To run pytest suite on model-archiver run following command
+
+```bash
+python -m pytest --cov-report html:htmlcov --cov=model_archiver/ model_archiver/tests/unit_tests/
+```
+
+The reports are generated at following path : `model-archiver/htmlcov/`
+
+## To run IT suite on model-archiver run following command
+
+```bash
+cd model-archiver
+pip install .
+python -m pytest --cov-report html:htmlcov --cov=model_archiver/ model_archiver/tests/integ_tests/
+```
 
-The reports can be accessed at the following paths:
+The reports are generated at following path : `model-archiver/htmlcov/`
```

Comment:
Whats the diff b/w the UT and IT coverage reports? Should these be written to two separate files, to avoid UT coverage being overwritten by IT coverage

Reply:
Done.